Sources/Core/API/Label.swift

```diff
@@ -153,10 +156,18 @@ public final class Label: SceneObject, InstanceHashable, ActionPerformer, Plugga
 
     private func fontDidChange() {
         layer.font = font
-        layer.fontSize = font.pointSize
+        layer.fontSize = font.pointSize * scale
```

Comment:
Instead of applying a multiplier to the font size, I think it's better to use the `scale` property on the label's layer. That way the actual rendered result gets scaled instead of the font. If we change the font size then we won't maintain the text's visual position if it's, for example, aligned to the left 🙂

Reply:
That actually makes sense :)

Reply:
One remark, @JohnSundell: If I scale the layer with `CATransform3DScale` the fontSize doesn't change, and the result (for upscaling) is pretty ugly:
<img width="367" alt="nofontsizechange" src="https://user-images.githubusercontent.com/48835/35071774-3e7af7aa-fbe2-11e7-9a41-58468d6a5d01.png">

Reply:
@Loyolny Yeah I think that's fine. Scaling is meant to do just that (it's how it works for Actor as well), it simply scales up the rendered result rather than changing the source material. It's really useful when you want to make some quick zoom-in effect or something like that 🙂 What do you think @mattiashagstrand?

Reply:
Fixed